src/lib.rs

```diff
@@ -669,13 +669,36 @@ impl Octocrab {
     ///# }
     /// ```
     pub async fn graphql<R: crate::FromResponse>(
+        &self,
+        payload: &(impl serde::Serialize + ?Sized),
+    ) -> crate::Result<R> {
+        self.post(
+            "graphql",
```

Comment:
@Libbum , due to switch from Url to Uri lib, the routes now require to be prefixed with `/`. I think I missed a few cases where I had to add `/`, and this seems to be one of them


Following should fix the issue
```suggestion
            "/graphql",
```

Reply:
Perfect, thanks! I'll give that a shot.